src/function/open-referral-function/src/FamilyHubs.OpenReferral.Function/Program.cs

```diff
@@ -0,0 +1,34 @@
+using FamilyHubs.OpenReferral.Function.ClientServices;
+using FamilyHubs.OpenReferral.Function.Repository;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+IHost host = new HostBuilder()
+    .ConfigureHostConfiguration(config =>
+    {
+        config.AddJsonFile("appsettings.json", optional: false, reloadOnChange: false);
+        config.AddJsonFile("appsettings.Development.json", optional: true, reloadOnChange: false);
+    })
+    .ConfigureFunctionsWebApplication()
+    .ConfigureServices(services =>
+    {
+        IConfiguration config = services.BuildServiceProvider().GetService<IConfiguration>()!;
+
+        services.AddApplicationInsightsTelemetryWorkerService();
+        services.ConfigureFunctionsApplicationInsights();
+
+        services.AddHttpClient<IHsdaApiService, HsdaApiService>(httpClient =>
+            httpClient.BaseAddress = new Uri(config["ApiConnection"]!)).SetHandlerLifetime(TimeSpan.FromMinutes(2));
```

Comment:
We can remove `.SetHandlerLifetime(TimeSpan.FromMinutes(2));` because the default is two minutes :)

https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.httpclientbuilderextensions.sethandlerlifetime?view=net-8.0

Reply:
Got rid here: https://github.com/DFE-Digital/fh-services/pull/63/commits/a670e1ca6ed706b8609f0a275bf1f2f07f9df8f4

I have a habit of setting stuff that's already default in-case the defaults ever change, bad habit 😄